node/src/mempool.rs

```diff
@@ -170,7 +215,12 @@ impl MempoolSrv {
 
             events.push(TransactionEvent::Included(tx));
             // Persist transaction in mempool storage
-            db.add_tx(tx)
+
+            let now = time::SystemTime::now()
+                .duration_since(UNIX_EPOCH)
+                .expect("valid timestamp");
+
+            db.add_tx(tx, now.as_secs())
```

Comment:
it's weird to see this passed as a parameter

Reply:
> Is there a specific reason for having timestamps passed as parameters instead of computing them inside add_tx and get_expired_txs?

- Make it explicit
- Keep DB methods lean

Any specific reason for having timestamps computed in add_tx / get_expired_txs?

Reply:
It's just weird to pass as a parameter something that is gonna be always the same.
Also this brings to duplication of code (e.g. to compute `now`).

Reply:
The param is also used in an unit test to facilitate the filtering out of subset of the txs.